commands/azureCommands/create-registry.ts

```diff
@@ -0,0 +1,133 @@
+    } catch (error) {
+        return;
+    }
+    const client = AzureCredentialsManager.getInstance().getContainerRegistryManagementClient(subscription);
+
+    let registryName: string;
+    try {
+        registryName = await acquireRegistryName(client);
+    } catch (error) {
+        return;
+    }
+
+    const sku: string = await vscode.window.showInputBox({
+        ignoreFocusOut: false,
+        placeHolder: 'Basic',
+        value: 'Basic',
+        prompt: 'SKU? '
+    });
+
+    client.registries.beginCreate(resourceGroup.name, registryName, { 'sku': { 'name': sku }, 'location': resourceGroup.location }).then(function (response) {
+        vscode.window.showInformationMessage(response.name + ' has been created succesfully!');
+    }, function (error) {
+        vscode.window.showErrorMessage(error.message);
+    })
+
+}
+
+// INPUT HELPERS
+async function acquireSubscription(): Promise<SubscriptionModels.Subscription> {
+    let subscription: SubscriptionModels.Subscription;
+    const subs = AzureCredentialsManager.getInstance().getFilteredSubscriptionList();
+
+    let subsNames: string[] = [];
+    for (let i = 0; i < subs.length; i++) {
+        subsNames.push(subs[i].displayName);
+    }
+    let subscriptionName: string;
+    do {
+        subscriptionName = await vscode.window.showQuickPick(subsNames, { 'canPickMany': false, 'placeHolder': 'Choose a subscription to be used' });
+
+        if (subscriptionName === undefined) throw 'User exit';
+    } while (!subscriptionName);
+
+
+    return subs.find(sub => { return sub.displayName === subscriptionName });
+}
+
+async function acquireResourceGroup(subscription: SubscriptionModels.Subscription): Promise<ResourceGroup> {
+    //Acquire each subscription's data simultaneously
+    let resourceGroup;
+    let resourceGroupName;
+    const resourceGroupClient = new ResourceManagementClient(AzureCredentialsManager.getInstance().getCredentialByTenantId(subscription.tenantId), subscription.subscriptionId);
+    let resourceGroups = await AzureCredentialsManager.getInstance().getResourceGroups(subscription);
+
+    let resourceGroupNames: string[] = [];
+    resourceGroupNames.push('+ Create new resource group');
+    for (let i = 0; i < resourceGroups.length; i++) {
+        resourceGroupNames.push(resourceGroups[i].name);
+    }
+
+    do {
+        resourceGroupName = await vscode.window.showQuickPick(resourceGroupNames, { 'canPickMany': false, 'placeHolder': 'Choose a Resource Group to be used' });
+        if (resourceGroupName === undefined) throw 'user Exit';
+        if (resourceGroupName === '+ Create new resource group') {
+            let opt: vscode.InputBoxOptions = {
+                ignoreFocusOut: false,
+                prompt: 'Resource group name? '
+            };
+            resourceGroupName = await vscode.window.showInputBox(opt);
+            let resourceGroupStatus: boolean = await resourceGroupClient.resourceGroups.checkExistence(resourceGroupName);
```

Comment:
Use the acquire resource group method from AzureCredentialsManager

Reply:
Should be fixed now